data/map planets.txt

```diff
@@ -4449,13 +4449,15 @@ planet Yedikule
 	security 0
 
 planet "Yniu Eiu"
+	attributes "rulei planet"
```

Comment:
I know the systems have "rulei" as well, but when you're running a location filter for attributes, you're either running against planets (source, destination, stopover) or systems (waypoint, on enter), so we don't need to specify in the attribute name whether it's for a planet or a system.
```suggestion
	attributes rulei
```